lib/puppet/type/cisco_bgp_neighbor_af.rb

```diff
@@ -384,6 +386,24 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end
 
+  newproperty(:prefix_list_in) do
+    desc 'prefix-list in state. Valid values are a string defining the name ' \
+         "of the prefix-list or 'default'."
+    munge do |value|
+      value = :default if value == 'default'
```

Comment:
Given that the next line casts `value` to a string, this line isn't needed is it?